clients/src/main/java/org/apache/kafka/clients/admin/AdminClient.java

```diff
@@ -229,6 +317,18 @@ public DescribeAclsResult describeAcls(AclBindingFilter filter) {
      *
      * This operation is supported by brokers with version 0.11.0.0 or higher.
      *
+     * <p>The following exceptions can be anticipated when calling {@code get()} on the futures obtained from the
+     * {@link DescribeAclsResult#values() values()} method of the  returned {@code DescribeAclsResult}</p>
+     * <ul>
+     *     <li>{@link org.apache.kafka.common.errors.SecurityDisabledException}
+     *     if security features are disabled</li>
```

Comment:
It might be better to be more specific here.  "if no authorizer is configured on the broker"